save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt

```diff
@@ -0,0 +1,53 @@
+package org.cqfn.save.backend.controllers
+
+import org.cqfn.save.backend.service.ProjectService
+import org.cqfn.save.entities.ProjectDto
+import org.cqfn.save.repository.GitRepository
+
+import org.slf4j.LoggerFactory
+import org.springframework.dao.DataAccessException
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+
+/**
+ * Controller to save project
+ *
+ * @property projectService service of project
+ * @property webClient webclient
+ */
+@RestController
+class CloneRepositoryController(
+    private val projectService: ProjectService,
+    private val webClient: WebClient,
+    ) {
+    private val log = LoggerFactory.getLogger(CloneRepositoryController::class.java)
+
+    /**
+     * Endpoint to save project
+     *
+     * @param projectDto information about project
+     */
+    @PostMapping(value = ["/cloneRepository"])
```

Comment:
I think this endpoint name is quite confusing; this is the main entry point from frontend, so it should be called somehing like `/submitExecutionRequest`